gui/build.rs

```diff
@@ -8,6 +9,30 @@ const LINUX_INCLUDE: &str = r#"
 "#;
 
 fn main() {
+    if std::env::var("CARGO_FEATURE_GUI_SLINT").is_ok() {
+        build_bin();
+    } else {
+        build_lib();
+    }
+}
+
+fn build_bin() {
+    // Compile installer
+    slint_build::compile_with_config(
+        "slint/installer.slint",
+        CompilerConfiguration::new().with_style(String::from("fluent-dark")),
+    )
+    .unwrap();
+
+    // Compile main
+    slint_build::compile_with_config(
+        "slint/main.slint",
+        CompilerConfiguration::new().with_style(String::from("fluent-dark")),
+    )
+    .unwrap();
+}
```

Comment:
Is it possible to use a wildcard here? Also I think it is better to rename the folder to the other name (e.g. `ui`) so we don't need to use `::slint` to reference the Slint crate.

Reply:
I renamed the module, but wildcards don't seems possible based on the the code: Maybe with some compile time magic

Reply:
In that case we can recursive list all files in the directory.

Reply:
I removed the other file. It seems you can have only one slint file (You can export multiple things, but only from one file)